src/test_application/main.cpp

```diff
@@ -420,12 +444,12 @@ void debug_print_machines()
         std::cout << "[" << int(ent) << "]: " << hier.m_name << "\n";
```

Comment:
`std::cout`

Reply:
Well this part of the code is commented out, and I forgot to re-implement it.

Reply:
Actually, this is a temporary CLI command. The spdlog headers would make it really hard to reads, any thoughts?